### PR TITLE
Ensure note matches example rendition of tts:textCombine example (#1128).

### LINKED ISSUE #1128: Text combine example is incorrect/misleading.
The note below the text combine example at https://www.w3.org/TR/ttml2/#style-attribute-textCombine implies that the render on the right uses different variants of the font for the affected characters. The latter however appear identical to the render on the left.

Suggest using the following render instead:

### PR DESCRIPTION
**Pull request by skynavga**
Closes #1128.

### CODE REVIEW

**Comment by skynavga** [comment]
@palemieux see https://github.com/w3c/ttml2/pull/1180/commits/9766653d5b4eeb0377a7718c26284e141d287a78

File: spec/ttml2.xml [hunk 1]
```diff
@@ -13465,9 +13465,9 @@ then a <loc href="#terms-presentation-processor">presentation processor</loc> mu
 </table>
 <p></p>
 <note role="elaboration">
-<p>In the above example, for the value <code>all</code> of <att>tts:textCombine</att>, the presentation processor selected half-width variants of the
- affected characters, but the resulting width is larger than the EM square of the surrounding vertical text, to avoid readability issues and as 
- permitted by TTML.</p>
+<p>In the above example, the rightmost region depicts the use of the <code>all</code> value, where
+the presentation processor has selected half-width variant glyphs for the affected characters, thus allowing the combined text to fit within
+same EM square as the surrounding vertical text.</p>
```
Comment (nigelmegitt) [inline spec/ttml2.xml]: Nit: s/same/the same size

**Comment by palemieux** [comment]
@nigelmegitt I approved the PR earlier.

**Comment by nigelmegitt** [comment]
Apologies @palemieux I don't know why I didn't see that.

**Pull request merged**